src/main/flight/pid_init.c

```diff
@@ -237,8 +237,13 @@ void pidInitFilters(const pidProfile_t *pidProfile)
 
 #ifdef USE_ACC
     const float k = pt3FilterGain(ATTITUDE_CUTOFF_HZ, pidRuntime.dT);
+    const float angleFeedforwardSmoothing = pidProfile->angle_feedforward_smoothing;
+    const float angleCutoffHz = 100.0f / angleFeedforwardSmoothing; // default of 50 -> 2.0Hz, 100 -> 1.0Hz etc
```

Comment:
You can make `angleFeedforwardSmoothing` a meaningful value by converting it to the time constant "tau".

```
cutoff = 1 / (2 * pi * tau)
```

So in your case for example

```
const float angleCutoffHz = 1000.0f / (2 * M_Pif * angleFFSmoothingWindowMs);  // average over the last windowMs milliseconds
```

Reply:
Okay, coming back to the last comment... I thought about the above a bit and if you want the PT1 to behave as close to a "normal" moving average with a particular number of samples `N`, you can get that by setting the cutoff of both filters to be equal.

The moving average has its cutoff at `f_cut = 1 / (2 * N * Ts)`
The PT1 has its cutoff at `f_cut = 1 / (2 * pi * tau)`

By setting the cutoff of the PT1 to that of the moving average we get a very similar behavior between the two.
```
1 / (2 * N * Ts) = 1 / (2 * pi * tau)
N * Ts = pi * tau
tau = N * Ts / pi
```

`N * Ts` is just the window size of the moving average (in seconds). Now we see the relationship between `tau` of the PT1 and the window size of the moving average. So at 8kHz a moving average with 8 samples has a window size of `8 * 125us = 1000us = 1ms`. So to make the PT1 very similar to a moving average, we just need to set the PT1 time constant `tau` to:
```
tau = windowSize / pi
```

To finally put this new finding into C code, we just substitute our "old" `tauMs´ with the new `windowSizeMs`.
```
const float angleCutoffHz = 1000.0f / (2 * pidProfile->angle_feedforward_windowMs);  // moving average over the last windowMs milliseconds
```
And this time we have a time window we can set via CLI that is very easy to grasp: We are approximately averaging all values over the last `angle_feedforward_windowMs` milliseconds using a PT1.

Reply:
Or to make it a bit shorter: by just multiplying the current `pidProfile->angle_feedforward_smoothing` values by 5, we can get a value that has actual "physical" meaning. The current default of 50 will become 250. This means by default we average all values over the last 250ms.